distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -51,6 +51,8 @@ org.codice.ddf.system.hostname=localhost
 org.codice.ddf.system.httpsPort=8993
```

Comment:
we should probably add a comment to the block above this section that sums up the difference between the different properties